src/contractTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/consumer/ccd/CcdCasePactTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.reform.wataskconfigurationapi.consumer.ccd;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.google.common.collect.Maps;
+import io.restassured.http.ContentType;
+import net.serenitybdd.rest.SerenityRest;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.wataskconfigurationapi.SpringBootContractBaseTest;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
```

Comment:
We need to also implement PACT tests for the ccd end points we use our WA tests. See SpringBootFunctionalBaseTest.

1. coreCaseDataApi.startForCaseworker(
            userToken,
            serviceToken,
            userInfo.getUid(),
            "IA",
            "Asylum",
            "startAppeal")

2. coreCaseDataApi.submitEventForCaseWorker(
            userToken,
            serviceToken,
            userInfo.getUid(),
            "IA",
            "Asylum",
            caseDetails.getId().toString(),
            true,
            submitCaseDataContent
        );